Shared/Features/FreakDetails/View/FreakDetailsView.swift

```diff
@@ -3,21 +3,65 @@ import SwiftUI
 struct FreakDetailsView: View {
     @Environment(\.presentationMode) var presentationMode
     var viewModel: FreakDetailsViewModel
+    // let freak = Bundle.main.decode([Freak].self, from: "MockData.json")
 
     var body: some View {
-        Text(viewModel.fullName)
-            .navigationBarBackButtonHidden(true)
-            .navigationBarItems(leading: Button(action: { self.presentationMode.wrappedValue.dismiss() }) {
-                HStack {
-                    Image(systemName: "chevron.left")
-                    Text(viewModel.fullName)
+        ZStack {
+            Rectangle()
+                .fill(Color("AccentColor"))
+                .ignoresSafeArea()
+                .frame(maxWidth: 350, maxHeight: 500, alignment: .bottom)
+                .cornerRadius(10)
+
+            VStack {
+                Image(viewModel.imageName)
+                    .resizable()
+                    .aspectRatio(contentMode: .fill)
+                    .frame(maxWidth: 160, maxHeight: 160)
+                    .cornerRadius(10)
+                    .accessibility(identifier: "freakImage")
+                Text(viewModel.fullName)
+                    .navigationBarBackButtonHidden(true)
+                    .navigationBarItems(leading: Button(action: { self.presentationMode.wrappedValue.dismiss() }) {
+                        HStack {
+                            Image(systemName: "chevron.left")
+                            Text(viewModel.fullName)
+                        }
+                    })
+
+                Text(viewModel.description)
+                    .padding()
```

Comment:
We need to make the text elements full width, to consider the case where we have fewer words on the description. Don't rely on the content to layout the view.